egui/src/widgets/drag_value.rs

```diff
@@ -262,21 +273,41 @@ impl<'a> Widget for DragValue<'a> {
                 .sense(Sense::click_and_drag())
                 .text_style(TextStyle::Monospace)
                 .wrap(false);
-            let response = ui.add_sized(ui.spacing().interact_size, button);
-            let response = response.on_hover_text(format!(
+
+            let mut hover_text = format!(
                 "{}{}{}\nDrag to edit or click to enter a value.",
                 prefix,
                 value as f32, // Show full precision value on-hover. TODO: figure out f64 vs f32
                 suffix
-            ));
+            );
+
+            if is_slow_speed {
+                hover_text.push_str("\nPress 'Shift' while dragging for better control.");
+            }
+
+            let response = ui.add_sized(ui.spacing().interact_size, button);
+            let response = response
+                .on_hover_cursor(CursorIcon::ResizeHorizontal)
+                .on_hover_text(hover_text);
 
             if response.clicked() {
                 ui.memory().request_focus(kb_edit_id);
                 ui.memory().drag_value.edit_string = None; // Filled in next frame
             } else if response.dragged() {
+                response.ctx.output().cursor_icon = CursorIcon::ResizeHorizontal;
+
                 let mdelta = response.drag_delta();
                 let delta_points = mdelta.x - mdelta.y; // Increase to the right and up
+
+                let is_slow_speed = self.is_slow_speed
+                    && response.ctx.input().modifiers.shift
+                    && !(response.ctx.input().modifiers.alt
+                        || response.ctx.input().modifiers.command);
```

Comment:
Add a helper to `Modifiers` instead so you can write `ui.input().modifiers.only_shift()` here

Reply:
> Thank you for the PR!
> 
> If I do this:
> 
> ```
> ui.add(egui::DragValue::new(scalar).speed(1.0).slow_speed(true));
> ```
> 
> Then dragging slowly still rounds to nearest integer (integer because speed=1). This can be easily fixed! The rounding is using `auto_decimals`, so when shift is pressed increase `auto_decimals` with 1.
> 
> I also think we should not have `slow_speed` as an option, but always enable it. It is a great feature, so let's not hide it! Or is there a reason why one would not want it?

Indeed when we work with integers slow speed is a bit redundant, but for controlling floats I think it is very useful.
I made it optional because I thought it would be a benefit to be able to opt-in for this behavior, but personally I think it should be always active.
I will update the PR first thing in the morning.